src/iosource/pcap/Source.cc

```diff
@@ -186,6 +186,13 @@ void PcapSource::OpenOffline()
 		InternalError("OS does not support selectable pcap fd");
 
 	props.link_type = pcap_datalink(pd);
+#ifdef DONT_HAVE_LIBPCAP_DLT_LINUX_SLL2
+        if ( props.link_type == 276 )
```

Comment:
There seems to be space vs tab confusion here and below.

Reply:
This is caught by the pre-commit checks which I just manually triggered in CI, see https://github.com/zeek/zeek/wiki/Coding-Style-and-Conventions#user-content-clang-format.

Reply:
Thanks @bbannier. Fixed in 7724b8e.

BTW, is there a guide somewhere for running the github actions and pre-commit hooks locally?

Reply:
As for the pre-commit GH action, we have the wiki paragraph I linked above (TLDR; `pre-commit run -a` to run all pre-commit hooks on all files in the repo). This automatically downloads e.g., a `clang-format` binary.

For the other GH actions see their definitions in `.github/workflows`.